internal/collector/parser.go

```diff
@@ -208,10 +208,16 @@ func (k *KeepalivedCollector) parseVRRPData(i io.Reader) ([]VRRPData, error) {
 
 			s := strings.Split(strings.TrimSpace(l), prop)
 			d.IName = strings.TrimSpace(s[1])
-		} else if strings.HasPrefix(l, "   ") && strings.Contains(l, prop) && d.IName != "" {
-			s := strings.Split(strings.TrimSpace(l), prop)
-			key := strings.TrimSpace(s[0])
-			val := strings.TrimSpace(s[1])
+		} else if strings.HasPrefix(l, "   ") && d.IName != "" {
+			key := ""
+			val := ""
+			if strings.Contains(l, prop) {
```

Comment:
I think this will be more reasonable:
```suggestion
			if !strings.Contains(l, prop) {
				continue
			}
```

Reply:
Yep, I've thought the same but under impression of the statement I've read in books something like "`if - then` must follow more possible way of execution" (then it would be cached in processor's cache ))

It seems the Golang [guide](https://golang.org/doc/effective_go.html#if) does not follow this statement)
Ok, I'll change this

Reply:
Yes. Thanks.